recipes/xpack/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "cci.20220223":
```

Comment:
Coul you ask to library author to make a new release? Last one is 1 year old: https://github.com/xyz347/xpack/releases

Reply:
I can try.

Reply:
I had the author release the version 1.0.2, I changed the conandata version to 1.0.2 as well